Makefile-deps.mk

```diff
@@ -21,6 +21,9 @@ else
 endif
 GO_CMD ?= go
 
+GOLANGCI_LINT = $(PROJECT_DIR)/bin/golangci-lint
+golangci-lint:
+	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.60.3
```

Comment:
These changes are necessary because I encountered an error with the previous version:

```
ERRO [linters_context/goanalysis] SA5012: panic during analysis: interface conversion: interface {} is nil, not *buildir.IR, goroutine 8413 [running]:
runtime/debug.Stack()
```